src/Arcus.Observability.Telemetry.AzureFunctions/Arcus.Observability.Telemetry.AzureFunctions.csproj

```diff
@@ -17,11 +17,11 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Think about this one, should we move it out or not as well or is it overkill given Azure Functions brings it in already @fgheysels ?

Reply:
Maybe here, we should also use the 3.1.8 packages. Diff issue perhaps. It's already too much in this PR for my opinon.

Reply:
The dependency to AspNetCore.App should be removed here indeed.   AFAIK, an Azure Function has no need to use the AspnetCore framework.

The Microsoft.Extensions.Logging dependency should just be added, regardless if you're targetting netstandard2.0 or netcoreapp3.1.   (I'll leave it up to you guys to decide which version you should use.     I'd think the lowest  version possible ?  )

Reply:
Th telemetry core package already has a 3.1.8 version package for the logging. Maybe we should stick with that version in the telemetry Azure Functions package.

Reply:
So, I fixed it's also in the telemetry core package changed. I saw that we didn't released this change with the 3.1.8 version everywhere; so we can change that.

Reply:
This also means that all projects that are using this Arcus library, will have to upgrade their version of the logging package if they're also explicitly using it.